addons/website/static/src/scss/website.scss

```diff
@@ -2446,3 +2446,43 @@ input[value*="data-oe-translation-initial-sha"] {
 .o_dropdown_without_offset {
     transform: none !important;
 }
+
+// Subtle input-group-text
+// This can be set to an input-group-text placed directly before or after the input.
+.input-group-text.input-group-text-subtle {
```

Comment:
Does it make sense to use both style in a same website? = Why is not this a an override of the `input-group-text` component itself?

Reply:
The aim is to having the possibility to have subtle elements when there’s no label and the classic `input-group-text` is too prominent. 

| Classic | Subtle |
|--------|--------|
| <img width="663" alt="Capture d’écran 2024-02-13 à 16 43 53" src="https://github.com/odoo/odoo/assets/80679690/40957612-6703-4388-b742-2517ecaeda48"> | <img width="658" alt="Capture d’écran 2024-02-13 à 16 43 30" src="https://github.com/odoo/odoo/assets/80679690/65c5eab8-e295-4066-8ff0-7000e7b11555"> | 

This can be combined with classic `input-group-text` if we want to highlight another decorative element for example.
<img width="656" alt="Capture d’écran 2024-02-13 à 15 35 43" src="https://github.com/odoo/odoo/assets/80679690/e0afb422-c61b-40fb-b33b-aac4ac77a9d3">

With this approach, we think it makes more sense for these subtle elements to be used only before and after the input.

Reply:
Ok, just wanted to confirm you really do think combining both is great :+1: